cmd/api/main.go

```diff
@@ -53,7 +53,7 @@ func NewServer(k8sClient kubernetes.Interface, controller routers.Controller) *S
 }
 
 func main() {
-	err := observability.Start()
+	err := observability.Start("kubearchive.api")
```

Comment:
I think `"kubearchive.api"` should be a constant instead of a magic string

Reply:
Moved to the top as a constant.